### ISSUE TITLE
broccoli ENOENT issue prevents use of this addon on Windows with ember >=2.7

### ISSUE THREAD

**Issue by arnebit** [mansona/ember-cli-notifications#100]
A while ago, I tried to update ember to 2.7 and could not run my application locally anymore. I went back to 2.6 and hoped the issue would magically go away, but trying 2.8 now changes little. 

The error message relates to broccoli processing ember-cli-notifications and includes the path "C:\C:\Users...", which may be the culprit. Removing this add-on makes 'ember s' succeed again, so I  guess this is the better place to report this than the ember issue tracker.

Reproduction: 
ember init
ember install ember-cli-notification
ember s

Result (the first line is ok and familiar):

```
Serving on http://localhost:4200/
The Broccoli Plugin: [CSSModules] failed with:
Error: ENOENT: no such file or directory, open 'C:\C:\Users\ABI\Code\ember-28-test\tmp\cssmodules-input_base_path-J7jK7ZXQ.tmp\0\modules\ember-cli-notifications\styles\globals.css'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at CSSModules.loadPath (C:\Users\ABI\Code\ember-28-test\node_modules\ember-cli-notifications\node_modules\ember-css-modules\node_modules\broccoli-css-modules\index.js:105:20)
...
```

Windows: 10 Pro
node: 4.4.7
npm: 3.10.8
ember-cli: 2.8.0

**Comment by m-nagy** [comment]
I have the same issue, v3.3.1 works fine on Ember 2.8.0 for me.
ember-cli-notifications 4.0.8 is not working.

**Comment by arnebit** [comment]
Yeah, sorry for not reporting back earlier, the issue seems indeed not to be with Ember 2.8.x but with the update to 4.0.x of this addon.

I went back to Ember 2.7.3 (also due to incompartibility with another module) and ember-cli-notifications 4.0.8 produces the above error, too, while 3.3.1 works fine and may have worked fine with Ember 2.8 as well (I do not quite remember).

**Comment by mrkongo** [comment]
Any workaround?

**Comment by ynnoj** [comment]
This error is stemming from an issue further up the dependency tree. I have filed an [issue](https://github.com/salsify/broccoli-css-modules/issues/1) with the guilty package. As soon as this is fixed, I'll do a subsequent release of the add-on.

Sorry for the inconvenience 😄

**Comment by ynnoj** [comment]
I just published `4.0.9` with updated dependencies that _should_ fix this issue.
